enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,340 @@
+  the Cloud Provider on a cluster that only supports TAT authentication with the Cloud Provider
+* As a cluster admin, I want to know which OLM Operators support tokenized authentication for my cloud, so that I can
+  provide token-based access for cloud resources for them.
+* As a cluster admin of a cluster using tokenized cloud auth, I want to know what's required to install and upgrade OLM 
+ Operators whenever those operators manage resources that authenticate against my cloud so they can function properly
+* As a cluster admin, I want the experience of TAT authentication to be as similar as possible from one Cloud Provider 
+  to the other, so that I can minimize Cloud Specific knowledge and focus more on OpenShift.
+* As an Operator developer, I want to have a standard framework to define tokenized authentication requirements and
+  consume them, per supported cloud, so that my operator will work on token-based authentication clusters.
+* As an Operator Hub browser, I want to know which operators support tokenized cloud auth and on which clouds so I can
+  see only a filtered list of operators that will work on the given cluster.
+* As an Operator Hub browser, I want to be informed / reminded in the UI that the cluster only supports TAT 
+  authentication with the Cloud Provider, so that I don't confuse the cluster with one that will try to mint long lived
+  credentials.
+* As an Operator Hub browser, I want to be able to easily provide what's required to the OLM operators I install through
+  the UI.
+* As the HyperShift team, where CCO is not installed so the only supported authentication mode is via TAT, I want the
+  Red Hat branded operators that must reach the Cloud Provider API, to be enabled to work with TAT credentials in a
+  consistent, and automated fashion so that customer can use those operators as easily as possible, driving the use of
+  layered products.
+
+### Goals
+
+Allow Day-2 Operators to access cloud provider resources as seamlessly as possible when being installed, used and
+updated on STS enabled clusters.
+
+While providing the above goal, allow for multi-tenancy. In this sense multi-tenancy means that an operator may
+enable its operands to communicate with the cloud provider instead of communicating with the cloud itself. Operands 
+would use the same set of credentials as the operator. The operator will need to maintain its own logic to minimize 
+conflicts when sharing credentials with operands.
+
+Operator authors have a way to notify, guide, and assist Day-2 Operator admins in providing the required cloud provider
+credentials matched to their permission needs for install and update.
+
+Ideally, a solution here will work in both HyperShift (TAT always) and non-HyperShift clusters.
+
+### Non-Goals
+
+Day 1 operators are not included in this proposal, they are pre-provisioned for TAT enabled clusters (at least 
```

Comment:
Aside from the work already being done, is there a compelling technical reason that the day-1 operators requiring TAT support would not use the same implementation?

Reply:
the might, just want to note it's fully out of scope for this EP